build/index.js

```diff
@@ -519,6 +520,11 @@ async function buildDocument(document, documentOptions = {}) {
     ? Number(metadata.popularity.toFixed(4))
     : 0.0;
 
+  if (doc.popularity) {
+    doc.popularityRanking =
+      1 + getAllPopularityValues().filter((p) => p > doc.popularity).length;
```

Comment:
do we do that for every document when building for prod? Does that affect build times?